Dll_Injector/main.cpp

```diff
@@ -6,9 +6,19 @@
 #define PROCESS_NAME "melonbooksviewer.exe"
 #define DLL_NAME "MelonDumper.dll"
 
+// usage: Dll_Injector.exe [-path <initial path to save image file>]
 int main(int argc, const char *argv[])
 {
-	std::string path = BrowseFolder();
+	const char* initialPath = "";
+	for (int index = 1; index < argc; ++index)
+	{
+		if (strcmp(argv[index], "-path") == 0 && index + 1 < argc)
```

Comment:
Single dash usually use in the short form of argument option, I think it is better to use two dash like `--path`.